### PR TITLE
quickstart for Mac

### PR DESCRIPTION
**Pull request by NatalieEO**
dockstore/dockstore#3570
Some of the formatting is weird inthe code, but looks like
Screen Shot 2020-06-17 at 12 27 34 PM

### CODE REVIEW

File: src/app/loginComponents/onboarding/downloadcliclient/downloadcliclient.component.ts [hunk 1]
```diff
@@ -80,17 +81,37 @@ Ensure that you are able to run Docker without using sudo directly with the
 sudo usermod -aG docker $USER
 exec newgrp docker
 \`\`\`
-3. Install the dockstore command-line program and add it to the path.
+
+#### Part 1 for MacOS - Install dependencies
+1. Install Java 11. Download the JDK from [AdoptOpenJDK](https://adoptopenjdk.net/index.html). Choose the MacOS Compressed Archive Version.
```
Comment (coverbeck) [inline src/app/loginComponents/onboarding/downloadcliclient/downloadcliclient.component.ts:86]: I install the JDK with HomeBrew. If you don't want to write that up in detail, I'd at least mention is as an alternative.

Comment (NatalieEO) [reply src/app/loginComponents/onboarding/downloadcliclient/downloadcliclient.component.ts]: Gave some instructions and also realized I left the wrong link in from this comment

File: src/app/loginComponents/onboarding/downloadcliclient/downloadcliclient.component.ts [hunk 2]
```diff
@@ -63,10 +64,10 @@ Setup our Dockstore CLI application to start launching workflows from the comman
 
 #### Requirements
 1. Linux/Ubuntu (Recommended - Tested on 18.04.3 LTS) or Mac OS X machine
-2. Java 11 (Tested with OpenJDK 11, Oracle JDK may work but is untested)
+2. Java 11 (Tested with OpenJDK 11 and Oracle JDK)
```
Comment (denis-yuen) [inline src/app/loginComponents/onboarding/downloadcliclient/downloadcliclient.component.ts:67]: This isn't right, we do not test with Oracle JDK. (at least not in the sense of unit/integration tests)

File: src/app/loginComponents/onboarding/downloadcliclient/downloadcliclient.component.ts [hunk 3]
```diff
@@ -80,17 +81,44 @@ Ensure that you are able to run Docker without using sudo directly with the
 sudo usermod -aG docker $USER
 exec newgrp docker
 \`\`\`
-3. Install the dockstore command-line program and add it to the path.
+
+#### Part 1 for MacOS - Install dependencies
+1. We'll cover two ways to install Java 11. One way is to download the JDK from [Oracle](https://www.oracle.com/java/technologies/javase/jdk11-archive-downloads.html). Choose the MacOS Compressed Archive Version and follow the commands below.
```
Comment (denis-yuen) [inline src/app/loginComponents/onboarding/downloadcliclient/downloadcliclient.component.ts:86]: For consistency, this should also use OpenJDK (like our existing tests, we will likely need to write Mac tests that use OpenJDK for licensing/philosophical reasons)

**Comment by denis-yuen** [comment]
> Nothing objectively wrong with this, but I'm personally not a fan of such lengthy instructions. 

Ditto, maybe this can be tabbed by OS (wonder if worthwhile detecting OS and displaying appropriate default)

File: src/app/loginComponents/onboarding/downloadcliclient/downloadcliclient.component.ts (hunk 2 above)
Comment (NatalieEO) [reply src/app/loginComponents/onboarding/downloadcliclient/downloadcliclient.component.ts]: Changed back

File: src/app/loginComponents/onboarding/downloadcliclient/downloadcliclient.component.ts (hunk 3 above)
Comment (NatalieEO) [reply src/app/loginComponents/onboarding/downloadcliclient/downloadcliclient.component.ts]: Using link to openJDK instead

File: src/app/loginComponents/onboarding/downloadcliclient/downloadcliclient.component.scss [hunk 4]
```diff
@@ -14,3 +14,7 @@
 .indent {
   margin-left: 40px;
 }
+
+.font {
```
Comment (garyluu) [inline src/app/loginComponents/onboarding/downloadcliclient/downloadcliclient.component.scss:18]: what happens without this?

Comment (NatalieEO) [reply src/app/loginComponents/onboarding/downloadcliclient/downloadcliclient.component.scss:18]: The font within the mat-tab-group is different. It looks bolded and the family is different (Roboto, "Helvetica Neue", sans-serif;)

Screen Shot 2020-06-18 at 10 59 48 AM

**Comment by coverbeck** [comment]
Verified instructions are present. Created dockstore/dockstore#3604 as a followup.

**Pull request merged**